### PR TITLE
Add documentation for how to get bors try privileges

### PR DESCRIPTION
**Pull request by radhikaj**
Also add workaround for Jenkins logs issue

Signed-off-by: Radhika Jandhyala <radhikaj@microsoft.com>

### CODE REVIEW

File: docs/Contributing.md [hunk 1]
```diff
@@ -169,8 +168,16 @@ available commands. The highlights:
...
 | bors retry | Run the previous command a second time.
 
+Anyone who has created atleast one PR in the Open Enclave repo may
+request `bors try` permissions. Please email
+oesdkcgc@lists.confidentialcomputing.io to request access.
+
 Also see our [Bors dashboard](https://app.bors.tech/repositories/21855/).
 
+After a bors run, there is a known issue where Jenkins builds don't have links
+to the build logs for each pipeline. Please see Issue#2878 for a work around for
```
Comment (mingweishih) [inline docs/Contributing.md:178]:
```suggestion
to the build logs for each pipeline. Please see #2878 for a work around for
```

File: docs/Contributing.md [hunk 2]
```diff
@@ -169,8 +168,16 @@ available commands. The highlights:
 | bors ping | Check if Bors is up. If it is, it will comment with _pong_.
 | bors retry | Run the previous command a second time.
 
+Anyone who has created atleast one PR in the Open Enclave repo may
```
Comment (CodeMonkeyLeet) [inline docs/Contributing.md:171]:
```suggestion
Anyone who has created at least one PR in the Open Enclave repo may
```

File: docs/Contributing.md [hunk 3]
```diff
@@ -139,7 +139,6 @@ Please do not:
 
 Merging Pull Requests
 ---------------------
-
```
Comment (CodeMonkeyLeet) [inline docs/Contributing.md:142]: Nit: Don't remove the extra space. [Markdownlint](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) recommends it, and it's consistent with the rest of this document.

File: docs/Contributing.md [hunk 4]
```diff
@@ -169,8 +168,16 @@ available commands. The highlights:
...
 | bors retry | Run the previous command a second time.
 
+Anyone who has created at least one PR in the Open Enclave repo may
+request `bors try` permissions. Please email
+oesdkcgc@lists.confidentialcomputing.io to request access.
+
 Also see our [Bors dashboard](https://app.bors.tech/repositories/21855/).
 
+After a bors run, there is a known issue where Jenkins builds don't have links
+to the build logs for each pipeline. Please see #2878 for a work around for
```
Comment (dthaler) [inline docs/Contributing.md:178]:
```suggestion
to the build logs for each pipeline. Please see #2878 for a workaround for
```

**Comment by radhikaj** [comment]
@dthaler @jhand2 Added a note about accepting suggestions and DCOing them. Please take a look.

**Comment by jhand2** [comment]
Machines running the build went down.

bors r+

**Pull request merged**